### PR TITLE
feat #26 added AdHoc Tool for getTypeAsShortString

### LINKED ISSUE #26: Add AdHoc Tool for getTypeAsShortString()
Implement the AdHoc coverage tool for the getTypeAsShortString function

### PR DESCRIPTION
**Pull request by agylling**
Fixes #26

- Type of change :
  - [x] New feature

### CODE REVIEW

File: karate-core/src/main/java/com/intuit/karate/ScriptValue.java [hunk 1]
```diff
@@ -70,36 +72,52 @@ public Object getValue() {
     }
 
     public String getTypeAsShortString() {
+        Boolean[] cov = AdhocCoverageTool.m.get("getTypeAsShortString");
         switch (type) {
             case NULL:
+                cov[1] = true;
```
Comment (gpihl) [inline karate-core/src/main/java/com/intuit/karate/ScriptValue.java]: Should be 0-indexed? What is this, MATLAB? :D

File: karate-core/src/main/java/com/intuit/karate/ScriptValue.java [hunk 2]
```diff
@@ -70,36 +72,52 @@ public Object getValue() {
...
                 return "null";
             case UNKNOWN:
+                cov[2] = true;
                 return "?";
             case PRIMITIVE:
+                Wcov[3] = true;
```
Comment (gpihl) [inline karate-core/src/main/java/com/intuit/karate/ScriptValue.java]: Wcov?

**Review by gpihl — CHANGES_REQUESTED** [review]

File: karate-core/src/main/java/com/intuit/karate/ScriptValue.java (hunk 2 above)
Comment (agylling) [reply karate-core/src/main/java/com/intuit/karate/ScriptValue.java]: Ye that was fixed before you noticed it :P Sorry :(

File: karate-core/src/main/java/com/intuit/karate/ScriptValue.java (hunk 1 above)
Comment (agylling) [reply karate-core/src/main/java/com/intuit/karate/ScriptValue.java]: Silly me xd

**Comment by agylling** [comment]
Fixed the misstakes :)

**Review by gpihl — APPROVED** [review]
LGTM! Need to resolve merge conflict though.

**Pull request merged**